js/pantry.js

```diff
@@ -31,75 +36,25 @@ function getLocalStorage(name) {
 
     return parsedArray;
   }
+  console.log(storageData);
 }
 
+
 function setLocalStorage(name, array) {
   const stringifiedArray = JSON.stringify(array);
   localStorage.setItem(name, stringifiedArray);
 }
 
 ////////////////// rendering functions
-//
-function renderTableButton(value, className, fn) {
-  let btn = document.createElement("input");
-  btn.type = "button";
-
-  btn.className = className;
-  btn.value = value;
-  btn.onclick = fn;
-  return btn;
-}
-
-function deleteItemFromTable(event) {
-  const th = event.target.parentNode;
-  const tr = th.parentNode;
-  tr.parentNode.removeChild(tr);
-  // re render the table?
-  // this function rerenders it automatically it seems, so don't need to RE render
-}
-
-function deleteItemFromStorage(array) {
-  // deletes an obj from Local Storage based off the name property
-  const currentLocalStorage = getLocalStorage("pantry");
-
-  const filtered = currentLocalStorage.filter(function (el) {
-    return el.name != array[0];
-  });
-
-  return filtered; // returns the new filtered array
-}
-
-function deleteItem(event) {
-  deleteItemFromTable(event);
-
-  const currentItemToDelete = event.target.parentNode.parentNode;
-  const currentItemTDs = currentItemToDelete.children;
-
-  // convert the HTMLCollection to an array
-  const newArray = [...currentItemTDs];
-  // remove the first item out of the array (this is the button in the table which we don't need right here)
-  newArray.shift();
-
-  const itemValues = [];
-  for (let td of newArray) {
-    itemValues.push(td.innerHTML);
-  }
-  setLocalStorage("pantry", deleteItemFromStorage(itemValues));
-
-  // delete obj
-  // delete item from local storage
-  // separate functions?
-}
-
```

Comment:
These should still be included in the file, not deleted. They were most likely added after you already pulled the code down. I'm trying to think of the best way to fix this.